.snippets/code/develop/application-devs/tooling/asset-transfer-api/overview/setup.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  AssetTransferApi,
+  constructApiPromise,
+} from "@substrate/asset-transfer-api";
+
+async function main() {
+  const { api, specName, safeXcmVersion } = await constructApiPromise(
+    "INSERT_WEBSOCKET_URL"
+  );
+
+  const assetsApi = new AssetTransferApi(api, specName, safeXcmVersion);
+
+  // Your code using assetsApi goes here
```

Comment:
I guess this is a typo, since I don't know anything named `assetsApi`

```suggestion
  // Your code using assetTransferApi goes here
```

Reply:
Yeah, I get that. My point here was that in some snippets, you initialized the variable in a plural form, and in others, you used assetApi in a singular